lib/dependencies/RequireEnsureDependenciesBlockParserPlugin.js

```diff
@@ -52,18 +73,24 @@ module.exports = class RequireEnsureDependenciesBlockParserPlugin {
 							if(failed) {
 								return;
 							}
-							if(fnExpression) {
-								if(fnExpression.fn.body.type === "BlockStatement")
-									parser.walkStatement(fnExpression.fn.body);
+							if(successExpression) {
+								if(successExpression.fn.body.type === "BlockStatement")
+									parser.walkStatement(successExpression.fn.body);
+								else
+									parser.walkExpression(successExpression.fn.body);
+							}
+							if(errorExpression) {
+								if(errorExpression.fn.body.type === "BlockStatement")
+									parser.walkStatement(errorExpression.fn.body);
 								else
-									parser.walkExpression(fnExpression.fn.body);
+									parser.walkExpression(errorExpression.fn.body);
 							}
 							old.addBlock(dep);
 						} finally {
 							parser.state.current = old;
 						}
-						if(!fnExpression) {
-							parser.walkExpression(fnExpressionArg);
+						if(!successExpression) {
+							parser.walkExpression(successExpressionArg);
 						}
```

Comment:
also walk the errorExpressionArg similar to the successExpressionArg.
maybe add a existence check